config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = ""
+baseURL = "https://opencue.io"
```

Comment:
Should this include `www` or doesn't matter?

Reply:
Currently `www` just redirects to this address, so this is the canonical address.